components/save-map/layer-synchronizer.service.ts

```diff
@@ -37,10 +39,23 @@ export class HsLayerSynchronizerService {
         element: lyr,
       });
     });
+    this.HsCommonLaymanService.authChange.subscribe((_) => {
+      this.reloadLayersOnAuthChange();
+    });
     this.crs = map.getView().getProjection().getCode();
     this.HsLaymanService.crs = this.crs;
   }
 
+  private async reloadLayersOnAuthChange() {
+    for (const layer of this.syncedLayers) {
+      const layerSource = layer.getSource();
+      layer.set('events-suspended', true);
+      layerSource.clear();
+      await this.pull(layer, layerSource);
```

Comment:
On logout this would return no features, right? If so we could maybe also remove layer from layermanager, or is it automatic through ol.layer remove event?

Reply:
Seems a bit too complicated. I would have to add the layer back on login afterwards. Also I'm populating the layer with some default values from anonymous 'browser' user when its not authorized yet.